grpc-testtool/proto/merkle/merkle.proto

```diff
@@ -55,48 +58,46 @@ message ViewHasResponse {
 }
 
 message ViewGetRequest {
-  uint32 view_id = 1;
+  uint32 id = 1;
   bytes key = 2;
 }
 
 message ViewGetResponse {
   bytes value = 1;
-  GetError err = 2;
 }
 
 message ViewNewIteratorWithStartAndPrefixRequest {
-  uint32 view_id = 1;
+  uint32 id = 1;
   bytes start = 2;
   bytes prefix = 3;
 }
 
 message ViewNewIteratorWithStartAndPrefixResponse {
-  uint32 iterator_id = 1;
+  uint32 id = 1;
+}
+
+message IteratorNextRequest {
+  uint32 id = 1;
+}
+
+message IteratorNextResponse {
+  repeated PutRequest data = 1;
```

Comment:
This should not be repeated, there is only one row returned, unless we plan to support batching responses?